apache2/modsecurity.c

```diff
@@ -240,6 +240,15 @@ int modsecurity_init(msc_engine *msce, apr_pool_t *mp) {
     return 1;
 }
 
+#ifdef WAF_JSON_LOGGING_ENABLE
+void modsecurity_handle_signals_for_reopen(int signum)
+{
+    if (signum == SIGUSR1) {
```

Comment:
This function is only registered to handle `SIGUSR1` so I guess we should not need this check here.

Reply:
changed